packages/cli/src/api/pseudoLocalize.js

```diff
@@ -45,18 +47,24 @@ const addDelimiters = R.compose(
   addDelimitersHTMLTags
 )
 
-function pseudoLocalize(message) {
-  pseudoloc.option.delimiter = delimiter
-  // We do not want prepending or appending because of Plurals structure
-  pseudoloc.option.prepend = ""
-  pseudoloc.option.append = ""
-  const pseudoLocalizeMessage = pseudoloc.str(message)
+function removeDelimiters(message) {
+  return message.replace(new RegExp(delimiter, "g"), "")
+}
 
-  return pseudoLocalizeMessage.replace(new RegExp(delimiter, "g"), "")
+export function setPseudoLocalizeOption() {
+  pseudolocale.option.delimiter = delimiter
+  // We do not want prepending and appending because of Plurals structure
+  pseudolocale.option.prepend = ""
+  pseudolocale.option.append = ""
+  isPseudoLocalizeOptionSet = true
 }
 
 export default function(message: string) {
+  if (!isPseudoLocalizeOptionSet) {
+    setPseudoLocalizeOption()
```

Comment:
This is probably slightly overengineered :) Since `pseudolocale` has global state, I think you would be fine setting those properties right after import without a need for special init flag. I guess it doesn't matter that much, it's just about clarity of the code.

Reply:
Done